mods/cnc/rules/aircraft.yaml

```diff
@@ -158,8 +158,15 @@ ORCA:
 	Explodes:
 		Weapon: HeliExplode
 		EmptyWeapon: HeliExplode
-	WithMoveAnimation:
-		MoveSequence: move
+	UpgradeOnMovement:
+		Upgrades: movingforward
+	WithFacingSpriteBody:
```

Comment:
Given our more recent discovery / discussion about animations and sprite bodies I think it might be a better idea to keep the Orca as it is, keeping this trait for downstream modders.